app/Http/Controllers/FarmerController.php

```diff
@@ -0,0 +1,77 @@
+<?php
+
+namespace App\Http\Controllers;
+use Inertia\Inertia;
+use Illuminate\Http\Request;
+use App\Models\User;
+use App\Models\Task;
+use Illuminate\Support\Facades\Auth;
+use Illuminate\Support\Facades\Log;
+use Illuminate\Support\Facades\Http;
+
+ class FarmerController extends Controller
+{
+    public function index(Request $request)
+    {   
+        $token = env('NEXT_PUBLIC_MAPBOX_TOKEN');
+        $user = Auth::user();
+        $user_id = $user->id;
+        $tasks = Task::withCount(['photos' => function ($query) {
```

Comment:
No need apis for migration and access data directly from database.

Reply:
What does this code do? The reason I am asking is if an API is already serving this purpose then we can use the api instead of rewriting the code here.

Reply:
Plan to use api for migration has been changed as there are lot of performance issues regarding loading data using apis with photos. Need to feed data directly in order to reduce the loading time.